src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,41 @@
+.nav {
+  height: 100%;
+
+  &__list {
+    display: flex;
+    gap: 20px;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    height: 100%;
+  }
+
+  &__link.is-active {
+    text-decoration: underline;
+    text-underline-offset: 22px;
+    color: $link-color;
+    border-radius: 8px;
+    text-decoration-thickness: 4px;
+  }
+
+  &__item {
+    display: flex;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    color: $text-color;
+    text-decoration: none;
+    height: 60px;
+
+    @include transition-mixin;
+  }
+
+  &__link:hover {
+    color: $link-color;
```

Comment:
Ensure that the variable `$link-color` is defined in your SCSS files. If it's not, this will cause a compilation error.